pages/reference/base-images/base-images.md

```diff
@@ -34,7 +34,11 @@ excerpt: Docker images maintained by {{ $names.company.lower }}
 ## How to Pick a Base Image
 
 <!-- TODO: Add image on how image hierarchy and discuss -->
-When starting out a project, it's generally easier to have a "fatter" image, which contains a lot of prebuilt dependencies and tools. These images help you get setup faster and work out the requirements for your project. For this reason, it's recommended to start with `-build` variants, and as your project progresses, switch to a `-run` variant with some [docker multistage build magic][multistage-build-docs] to slim your deploy image down. In most cases, your project can just use a Debian based distribution, which is the default if not specified, but if you know the requirements of your project or prefer specific distros, Ubuntu, Alpine, and Fedora images are available. The set of `balenalib` base images follow a simple naming scheme described below, which will help you select a base image for your specific needs.
+When starting out a project, it's generally easier to have a "fatter" image, which contains a lot of prebuilt dependencies and tools. These images help you get setup faster and work out the requirements for your project. For this reason, it's recommended to start with `-build` variants. Also, images whose name begins with `%%BALENA_MACHINE_NAME%%` generally include more tools, while `%%BALENA_ARCH%%` named images are kept as small as possible.
```

Comment:
Should we include some examples of the differences? I assume you referring to the extra packages installed in the device-type base images like this?
https://github.com/balena-io-library/base-images/blob/master/balena-base-images/device-base/raspberrypi3/debian/bullseye/build/Dockerfile

Reply:
How about some stats as well. Everyone likes stats of how big/small are the images.